.github/workflows/gh-actions.yml

```diff
@@ -39,13 +41,16 @@ jobs:
       - run: sudo apt update
       - run: sudo apt install gcc-multilib
       - run: $RUNNER -q --jerry-tests --buildoptions=--compile-flag=-m32,--cpointer-32bit=on
+      - run: $RUNNER -q --jerry-tests --buildoptions=--compile-flag=-m32,--cpointer-32bit=on --build-debug
 
   OSX_x86-64_Build_Correctness_Unit_Tests:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
-      - run: $RUNNER -q --jerry-tests
+      - run: $RUNNER -q --unittests --build-debug
+      - run: $RUNNER -q --jerry-tests --build-debug
       - run: $RUNNER -q --unittests
+      # FIXME deadloop - run: $RUNNER -q --jerry-tests
```

Comment:
We already run jerry-tests on OSX without deadloop. What changed with this PR?

Reply:
It's depends on this
https://github.com/jerryscript-project/jerryscript/pull/4530

Reply:
I saw #4530 . But how is it possible that we haven't noticed it yet on the CI. Now there is no deadloop on the CI on OSX job. Could you describe what configuration changed, that triggered this issue?